store/testdata/search-test-fixture.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
Suggestion to split the sample data and testable. This is to make it more consistent with the other sample data and to hopefully make it easier to understand the unit test, since separating them will make it more clarity.